### PR TITLE
Fixed config.plist so it opens in Clover Configurator

### PR DESCRIPTION
**Pull request by lucacri**
Changed some characters so that the config.plist can be opened in clover configurator

### CODE REVIEW

**Review by korzhyk — CHANGES_REQUESTED** [review]
`ScreenResolution` not a good. I'm using a QHD monitor size so GUI will be blurred. Need to remove this, Clover in most cases detects a native resolution. Also keep comments in `Tableid` keys, because BIOS provide tables with name `ALASKA` for all tables and in this case we can drop tables only by length.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
 * BIOS Setting EHCI Hand-Off must be enabled
 * Fresh Clover with generated SMBIOS.plist *this file contains serial number so you must [genenerate][1] your own*
 
-# Installation
+## BIOS Settings
+
+Make sure you upgrade your motherboard BIOS to the version F7
+
+* *Save & Exit* -> Load Optimized Defaults
+* *M.I.T.* -> Advanced Memory Settings  Extreme Memory Profile(X.M.P.) : **Profile1**
```
Comment (korzhyk) [inline README.md:29]: Only if you have XMP profile support by your RAM.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
+* *BIOS* -> Fast Boot : **Disabled**
+* *BIOS* -> LAN PXE Boot Option ROM : **Disabled**
+* *BIOS* -> Storage Boot Option Control : **UEFI**
```
Comment (korzhyk) [inline README.md:32]: Is necessary when is CSM is disabled.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,52 @@
...
+* *Peripherals* -> Trusted Computing -> Security Device Support : **Disable**
+* *Peripherals* -> Network Stack Configuration -> Network Stack : **Disabled**
+* *Peripherals* -> USB Configuration -> Legacy USB Support : **Auto**
+* *Peripherals* -> USB Configuration -> XHCI Hand-off : **Enabled**
+* *Chipset* -> Vt-d : **Disabled**
```
Comment (korzhyk) [inline README.md:37]: Is necessary, we drop DMAR table via Clover and I'm using Vt-d in Windows.

File: README.md [hunk 4]
```diff
@@ -1,33 +1,52 @@
...
+* *Chipset* -> Wake on LAN Enable : **Disabled**
```
Comment (korzhyk) [inline README.md:38]: Why not?

File: README.md [hunk 5]
```diff
@@ -1,33 +1,52 @@
...
+
+Make sure you upgrade your motherboard BIOS to the version F7
+
+* *Save & Exit* -> Load Optimized Defaults
+* *M.I.T.* -> Advanced Memory Settings  Extreme Memory Profile(X.M.P.) : **Profile1**
+* *BIOS* -> Fast Boot : **Disabled**
+* *BIOS* -> LAN PXE Boot Option ROM : **Disabled**
+* *BIOS* -> Storage Boot Option Control : **UEFI**
+* *Peripherals* -> Trusted Computing -> Security Device Support : **Disable**
+* *Peripherals* -> Network Stack Configuration -> Network Stack : **Disabled**
```
Comment (korzhyk) [inline README.md:34]: Any issues?

File: README.md (hunk 1 above)
Comment (lucacri) [reply README.md:29]: Yes this should be optional

File: README.md (hunk 2 above)
Comment (lucacri) [reply README.md:32]: Sorry I'm new to this board. What do you mean?

File: README.md (hunk 3 above)
Comment (lucacri) [reply README.md:37]: This is all taken from https://hackintosher.com/builds/gigabyte-z370-aorus-gaming-7-hackintosh-build-guide/ as well as I'm running it. What happens if we leave it enabled? Works?

File: README.md (hunk 4 above)
Comment (lucacri) [reply README.md:38]: This might interfere with PowerNap (I can't get it to work)

File: README.md (hunk 5 above)
Comment (lucacri) [reply README.md:34]: No, but it works with this settings (gotten from https://hackintosher.com/builds/gigabyte-z370-aorus-gaming-7-hackintosh-build-guide/ ) so I never changed them

File: README.md (hunk 1 above)
Comment (lucacri) [reply README.md:29]:
```suggestion
* *M.I.T.* -> Advanced Memory Settings  Extreme Memory Profile(X.M.P.) : **Profile1** (optional)
```

File: OEM/Z370 AORUS Gaming 7/config.plist [hunk 6]
```diff
@@ -144,6 +144,10 @@
   </array>
   <key>GUI</key>
   <dict>
+    <key>ScreenResolution</key>
```
Comment (lucacri) [inline OEM/Z370 AORUS Gaming 7/config.plist]:
```suggestion
    
```

File: OEM/Z370 AORUS Gaming 7/config.plist [hunk 7]
```diff
@@ -144,6 +144,10 @@
...
+		<string>1400x1050</string>
```
Comment (lucacri) [inline OEM/Z370 AORUS Gaming 7/config.plist]:
```suggestion
		
```

File: OEM/Z370 AORUS Gaming 7/config.plist [hunk 8]
```diff
@@ -78,15 +78,15 @@
         <string>DMAR</string>
       </dict>
       <dict>
-        <key>#TableId</key>
+        <key>TableId</key>
```
Comment (lucacri) [inline OEM/Z370 AORUS Gaming 7/config.plist]: If we keep the # at the beginning, the file won't open in Clover Configurator. It's not a big deal, but are you sure it's necessary to keep the `#`?

File: README.md (hunk 3 above)
Comment (korzhyk) [reply README.md:37]: Yes, with no problems.

File: README.md (hunk 2 above)
Comment (korzhyk) [reply README.md:32]: When CSM Support is disabled BIOS will use only UEFI bootloaders.

**Comment by korzhyk** [comment]
And **remember** do not use Clover Configurator!

File: README.md (hunk 2 above)
Comment (lucacri) [reply README.md:32]: So 

*BIOS* -> Storage Boot Option Control : **UEFI**

is if you only want a Mac with single boot?

File: OEM/Z370 AORUS Gaming 7/config.plist (hunk 8 above)
Comment (korzhyk) [reply OEM/Z370 AORUS Gaming 7/config.plist]: You can change `#` to `_`, but using a Clover Configurator it's not good idea!

File: OEM/Z370 AORUS Gaming 7/config.plist [hunk 9]
```diff
@@ -144,6 +144,10 @@
   </array>
   <key>GUI</key>
   <dict>
+    <key>ScreenResolution</key>
+		<string>1400x1050</string>
+		<key>Theme</key>
+		<string>embedded</string>
```
Comment (korzhyk) [inline OEM/Z370 AORUS Gaming 7/config.plist]: It's a default value. You can omit this.

File: OEM/Z370 AORUS Gaming 7/config.plist (hunk 8 above)
Comment (lucacri) [reply OEM/Z370 AORUS Gaming 7/config.plist]: why not? (again, new to this, sorry!)

**Comment by lucacri** [comment]
Does your system work with PowerNap? in my case if I put it to sleep, it turns off for a second, then goes on (fans blasting at 100%), screen stays off, and that's it.

Is yours able to actually go in Power Nap and stay "off"?

File: README.md (hunk 2 above)
Comment (korzhyk) [reply README.md:32]: No, All moders OS' use a `UEFI` boot loaders not legacy `MBR`.

Comment (lucacri) [reply README.md:32]: Yeah I figured as much, I'm trying to understand what was the "CSM" part you are referring to

Comment (korzhyk) [reply README.md:32]: I'm mean that options `LAN PXE Boot Option ROM` and `Storage Boot Option Control` are legacy and not active when CSM Support is disabled.

File: OEM/Z370 AORUS Gaming 7/config.plist (hunk 8 above)
Comment (korzhyk) [reply OEM/Z370 AORUS Gaming 7/config.plist]: CC (Clover Configurator) not maintained by Clover developers. In same cases CC can corrupt config file. I'm using a PlistEdit Pro in macOS and Sublime Text on Windows. To work with base64 and another binary stuff I use https://goo.gl/tz7scz

**Comment by korzhyk** [comment]
@lucacri Currently I can't check sleep in Mojave. In HS sleep works great except issue 1 https://github.com/korzhyk/CLOVER-Z370-AORUS-Gaming-7#issues

File: OEM/Z370 AORUS Gaming 7/config.plist (hunk 8 above)
Comment (korzhyk) [reply OEM/Z370 AORUS Gaming 7/config.plist]: Or you can change `#TableId` to `Comment` its will be great to know what is goes to drop.
```suggestion
        <key>Comment</key>
```

File: OEM/Z370 AORUS Gaming 7/config.plist [hunk 10]
```diff
@@ -105,7 +105,7 @@
   <key>Boot</key>
   <dict>
     <key>Arguments</key>
-    <string>ngfxcompat=1</string>
+    <string>ngfxcompat=1 -v</string>
```
Comment (korzhyk) [inline OEM/Z370 AORUS Gaming 7/config.plist]: The `verbose` flag I'm using for testing purposes only. If you need to verbose output you can choose some flags in Clover. When you select boot entry and press `space-bar` you can edit boot args.

**Comment by Netskymm** [comment]
Hi guys, are there any solutions for sleep / waking up, without ejecting usb drives?

**Comment by korzhyk** [comment]
> Hi guys, are there any solutions for sleep / waking up, without ejecting usb drives?

Did you overclock your memory too? This issue only affected when I overclock my RAM but if using a XMP profile there are no issues with `not properly ejected USB`

**Pull request closed without merging**